mscore/inspector/inspector.cpp

```diff
@@ -851,32 +851,8 @@ InspectorClef::InspectorClef(QWidget* parent)
 
 void InspectorClef::setElement()
       {
-      otherClef = nullptr;                      // no 'other clef' yet
+      otherClef = toClef(inspector->element())->otherClef();
```

Comment:
Does `toClef()` always return non null ptr here?

Reply:
No...

Reply:
I believe the question is about whether it is safe to call `otherClef()` on the result of `toClef(inspector->element())`. Judging by the code that was there, it would appear that this is indeed safe. And `inspector->element()` is valid even before the call to `InspectorElementBase::setElement()`.